index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+<h1>How Blue Jeff Goldblum?</h1>
+		<!-- card number one -->
+	<article class="card">
+		<header class="cardTitle">
+			<h1>Blue</h1>  
+		</header>	
+			<section class="test"> <!-- image section-->
+				<div class="overlay"><div class="blue"></div></div>
+			  <div class="image"><img src="img.jpg" alt="Blue" height="225.204" width="150"></div>
+				<div class="description"> 
+					This is a top tier product. The delux Blue Jeff Goldblum will surely meet your everyday needs. You should definitely purchase this award wining and highly "featured" product. Go ahead and add it to your shopping cart. You'll be mad if you don't!
+				</div>     <!-- description -->
+				<div class="avalibility"> Avalible </div>     <!-- availability -->
+			</section> 
+			<section class="specs">      <!-- specifications -->
+				<header> Specifications </header> 
+				<div>&nbsp&nbspSize: 12' x 12'</div>     <!-- size -->
```

Comment:
It's bad practice to use non-breaking spaces.  If you need some room at the beginning of an element use CSS to add a margin or padding.